cli/exchange/node.go

```diff
@@ -180,6 +185,10 @@ func NodeCreate(org, nodeIdTok, node, token, userPw, arch string, nodeName strin
 		} else {
 			msgPrinter.Printf("Node %v created.", nodeId)
 			msgPrinter.Println()
+			if strings.Contains(resp.Msg, "Warning: you are within") {
+				msgPrinter.Printf("Note: the org %v has reached to the 95%% of the node limit.", org)
```

Comment:
Please change the text to "Note: the org %v has reached 95%% of the node limit: %v.", org, nodeLimit
I think you will have to call GET on the exchange  /orgs/{org} resource to get the limit, if the limit is not in the exchange message.

Reply:
Do not worry about the limits here, the exchange-api will change the text. It will have the limit in the message.

Reply:
What is the new message from exchange-api? Maybe it will help to resolve both issues.